graph/typeDefs.graphql

```diff
@@ -884,6 +912,54 @@ input RejectUsernameInput {
   message: String!
 }
 
+# Configurable settings that can be overridden for the Asset.
+input AssetSettingsInput {
```

Comment:
Is it possible to only change a subset of Asset Settings?

Reply:
What happens if I only provide a subset of the fields

Reply:
I've updated the comment to reflect the behaviour. You should update all fields when posting, you cannot specify a subset. This is because for some fields, the `null` value is used.